demo/components-mix-demo.html

```diff
@@ -0,0 +1,296 @@
+             overlay-description="Overlay description text : Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam porta sem malesuada magna mollis euismod. Cras justo odio, dapibus ac facilisis in, egestas eget quam. Praesent commodo cursus magna, vel scelerisque nisl consectetur et. Cras justo odio, dapibus ac facilisis in, egestas eget quam. Maecenas sed diam eget risus varius blandit sit amet non magna eget lacinia odio sem nec elit. Donec sed odio dui.  Etiam porta sem malesuada magna mollis euismod. eget lacinia odio sem nec elit. Donec sed odio dui Etiam porta sem malesuada magna mollis euismod."
+             title-action-button='{"key":"103","size":"btn--small","type":"btn--icon","icon":"px-nav:reload"}'
+             action-buttons='{"items":[{"key":"1","size":"btn--small","type":"btn--bare btn--icon","icon":"px-utl:link"},{"key":"2","size":"btn--small","type":"btn--bare btn--icon","icon":"px-utl:copy"},{"key":"3","size":"btn--small","type":"btn--bare btn--icon","icon":"px-utl:download"}]}'
+             hoverable>
+        <div slot="thumbnail" class="px-vis-pie-chart"><px-vis-pie-custom-component></px-vis-pie-custom-component></div>
+    </px-tile>
+    <px-tile 
+             title="13-88937"
+             subtitle="7E 01 Stage 1 Bucket"
+             description="Duis mollis, est non commodo luctus, nisi erat porttitor ligula, eget lacinia odio sem nec elit. Donec sed odio dui. Duis mollis, est non commodo luctus, nisi erat porttitor ligula, eget lacinia odio sem nec elit. Donec sed odio dui. Duis mollis, est non commodo luctus, nisi erat porttitor ligula, eget lacinia odio sem nec elit. Donec sed odio dui."
+             overlay-description="Overlay description text : Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam porta sem malesuada magna mollis euismod. Cras justo odio, dapibus ac facilisis in, egestas eget quam. Praesent commodo cursus magna, vel scelerisque nisl consectetur et. Cras justo odio, dapibus ac facilisis in, egestas eget quam. Maecenas sed diam eget risus varius blandit sit amet non magna."
+             title-action-button='{"key":"104","size":"btn--small","type":"btn--icon","icon":"px-nav:favorite"}'
+             action-buttons='{"items":[{"key":"1","val":"Return","icon":"px-utl:edit","selected":true},{"key":"2","val":"Scrap","icon":"px-utl:copy"},{"key":"3","val":"Remove","icon":"px-vis:trash-series"},{"key":"4","val":"Notification","icon":"px-nav:notification"}],"sortMode":"key","buttonStyle":"icon","icon":"px-nav:more","displayValue":"Select","disabled":false,"disableClear":false,"hideChevron":true,"multi":false,"searchMode":false}'
```

Comment:
The overflow menu (...) icon is really dark in this demo instead of changing to white like some of the other examples. Not sure where in the code/style that's a problem but I'm chucking this here. :)